packages/react-native-bpk-component-nudger/src/BpkNudger-test.common.js

```diff
@@ -93,52 +92,6 @@ const commonTests = () => {
         .toJSON();
       expect(tree).toMatchSnapshot();
     });
-
-    it('should support theming', () => {
-      const onPressFn = jest.fn();
-      const theme = {
-        buttonSecondaryTextColor: 'red',
-        buttonSecondaryBackgroundColor: 'yellow',
-        buttonSecondaryBorderColor: 'blue',
-      };
-      const tree = renderer
-        .create(
-          <BpkThemeProvider theme={theme}>
-            <BpkNudger
-              min={1}
-              max={9}
-              value={3}
-              decreaseButtonLabel="Decrease"
-              increaseButtonLabel="Increase"
-              onChange={onPressFn}
-            />
-          </BpkThemeProvider>,
-        )
-        .toJSON();
-      expect(tree).toMatchSnapshot();
-    });
-
-    it('should disable theming if the required attribute is omitted', () => {
-      const consoleErrorFn = jest.fn();
-      jest.spyOn(console, 'error').mockImplementation(consoleErrorFn);
-      const theme = {};
-      const tree = renderer
-        .create(
-          <BpkThemeProvider theme={theme} type="primary">
-            <BpkNudger
-              min={1}
-              max={9}
-              value={3}
-              decreaseButtonLabel="Decrease"
-              increaseButtonLabel="Increase"
-              onChange={() => null}
-            />
-          </BpkThemeProvider>,
-        )
-        .toJSON();
-      expect(tree).toMatchSnapshot();
-      expect(consoleErrorFn.mock.calls.length).toBe(1);
-    });
```

Comment:
These tests broke, and when I went to fix them I realised that they're redundant because they're testing theming on BpkButton, so I don't think that's something BpkNudger should worry about (the theming is a side effect of using button).